xml/System.Windows.Media/ScaleTransform.xml

```diff
@@ -24,7 +24,7 @@
 ## Remarks  
  Use a <xref:System.Windows.Media.ScaleTransform> to stretch or shrink an object horizontally or vertically.  The <xref:System.Windows.Media.ScaleTransform.ScaleX%2A> property specifies by how much to stretch or shrink an object along the x-axis, and the <xref:System.Windows.Media.ScaleTransform.ScaleY%2A> property specifies by how much to stretch or shrink an object along the y-axis. Scale operations are centered on the point specified by the <xref:System.Windows.Media.ScaleTransform.CenterX%2A> and <xref:System.Windows.Media.ScaleTransform.CenterY%2A> properties.  
   
- **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.ScaleTransform> class provides several special features: <xref:System.Windows.Media.ScaleTransform> objects can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.ScaleTransform> class provides several special features: <xref:System.Windows.Media.ScaleTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.ScaleTransform> class provides several special features: <xref:System.Windows.Media.ScaleTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```